src/Worlds.js

```diff
@@ -129,6 +141,83 @@ const worlds: {|
     'Sketchpad': WorldProperties,
     'Space': WorldProperties
 |} = {
+    'AmusementPark': {
+        background: AmusementPark,
+        backgroundGray: AmusementParkGray,
+        backgroundContrast: AmusementParkContrast,
+        backgroundInfo: {
+            A1: 'entrance',
+            C1: 'gameBooth',
+            D1: 'gameBooth',
+            G1: 'pirateShip',
+            H1: 'pirateShip',
+            J1: 'swingRide',
+            K1: 'swingRide',
+            L1: 'swingRide',
+            A2: 'entrance',
+            C2: 'gameBooth',
+            D2: 'gameBooth',
+            G2: 'pirateShip',
+            H2: 'pirateShip',
+            J2: 'swingRide',
+            K2: 'swingRide',
+            L2: 'swingRide',
+            A3: 'merryGoRound',
+            B3: 'merryGoRound',
+            G3: 'pirateShip',
+            H3: 'pirateShip',
+            A4: 'merryGoRound',
+            B4: 'merryGoRound',
+            E4: 'waterSlide',
+            F4: 'waterSlide',
+            J4: 'goKart',
+            K4: 'goKart',
+            L4: 'goKart',
+            E5: 'waterSlide',
+            F5: 'waterSlide',
+            G5: 'ferrisWheel',
+            H5: 'ferrisWheel',
+            I5: 'ferrisWheel',
+            J5: 'goKart',
+            K5: 'goKart',
+            L5: 'goKart',
+            A6: 'rollerCoaster',
+            B6: 'rollerCoaster',
+            D6: 'waterPark',
+            E6: 'waterPark',
+            F6: 'whaleFountain',
+            G6: 'ferrisWheel',
+            H6: 'ferrisWheel',
+            I6: 'ferrisWheel',
+            J6: 'goKart',
+            K6: 'goKart',
+            A7: 'rollerCoaster',
+            B7: 'rollerCoaster',
+            C7: 'rollerCoaster',
+            G7: 'ferrisWheel',
+            H7: 'ferrisWheel',
+            I7: 'ferrisWheel',
+            K7: 'snackStand',
+            L7: 'snackStand',
+            A8: 'rollerCoaster',
+            B8: 'rollerCoaster',
+            C8: 'rollerCoaster',
+            D8: 'rollerCoaster',
+            K8: 'snackStand',
+            L8: 'snackStand'
+        },
+        thumbnail: AmusementParkThumbnail,
+        thumbnailDark: AmusementParkThumbnail,
+        thumbnailGray: AmusementParkThumbnailGray,
+        thumbnailContrast: AmusementParkThumbnailContrast,
+        character: AmusementParkTicket,
+        characterGray: AmusementParkTicketGray,
+        characterContrast: AmusementParkTicketContrast,
+        startingX: 1,
+        startingY: 2,
+        startingDirection: 2, // East
+        enableFlipCharacter: true
```

Comment:
It's pretty subtle and small but I think it feels a little better with `enableFlipCharacter: false`